plugins/wacom-usb/fu-wac-device.c

```diff
@@ -945,6 +957,9 @@ fu_wac_device_set_progress(FuDevice *self, FuProgress *progress)
 static gchar *
 fu_wac_device_convert_version(FuDevice *device, guint64 version_raw)
 {
+	if (version_raw > G_MAXUINT16)
+		return fu_version_from_uint32(version_raw, fu_device_get_version_format(device));
+
```

Comment:
We've not got any coverage for this -- can we get emulation "enumeration-only" for a device with this version2 data set please? i.e. follow https://github.com/fwupd/fwupd/blob/main/docs/device-emulation.md#add-emulation-only-data-to-the-plugin and save the two new json files to `plugins/wacom-usb/tests/wacom-THENEWDEVICE-setup.json` and `plugins/wacom-usb/tests/wacom-THENEWDEVICE.json` and finally add them to `plugins/wacom-usb/meson.build` too. Or if you upload the emulation.zip to this bug I can do it too.

Reply:
I attached the file as emulation-wacom.zip. Would you do further setup?
Since it includes only one json file in the zip and no instruction how to add the json file to meson.build at
the instruction page, I'm not even sure if I'm correctly following the process to test it.
[emulation-wacom.zip](https://github.com/user-attachments/files/17693976/emulation-wacom.zip)

Reply:
Just in case, I took the emulation data by conducting reinstall of the cab file into the device.
[wacom-movink-reinstall.zip](https://github.com/user-attachments/files/17695367/wacom-movink-reinstall.zip)

Reply:
@flying-elephant can you cherry pick 1099acbaf0585fbfd6233bfddf1b841f515d93c4 onto your branch please. :)

Reply:
I updated the branch with your change. What should I do next?

I tried to do "fwupdmgr device-emulation". The result seems to miss some item as the output describes:
`(fwupd) tats@fedora:~/0_Program/fwupd/fwupd_commit_version/fwupd$ fwupdmgr device-emulate ./plugins/wacom-usb/tests/wacom-movink-13.json
Missing or outdated PolicyKit policy in /usr/share/polkit-1/actions/org.freedesktop.fwupd.policy
Copy into filesystem? [y/N]
y
[sudo] password for tats: 
WARNING: This package has not been validated, it may not work properly.
failed to load @enumeration_datadir@/wacom-movink-13-setup.json: failed to open @enumeration_datadir@/wacom-movink-13-setup.json
(fwupd) tats@fedora:~/0_Program/fwupd/fwupd_commit_version/fwupd$ 
`

Reply:
> plugins/wacom-usb/tests/wacom-movink-13.json

That's the one that has the @enumeration_datadir@ substitution -- if you use the `data/device-tests/wacom-movink-13.json` in the builddir it should work fine.

Reply:
Thank you!
It looks like I got it worked although what the log shows is only 
[emulation-only_movink_log.txt](https://github.com/user-attachments/files/17724886/emulation-only_movink_log.txt)
"OK".
Do you need any other file that you want after the completion of the device emulation for this particular fix?